### PR TITLE
Add some more examples and info to the README

### PR DESCRIPTION
**Pull request by nrfox**
Adds a `configuration` and `troubleshooting` section to the README to give more explanation on the worker config and how to run terraform commands for a definition.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -62,6 +62,52 @@ Build and publish the package to PYPI:
...
+
+A project is configured through a worker config, a yaml file that specifies the definitions, inputs, outputs, providers and all other necessary configuration. The worker config is what specifies how state is shared among your definitions. The config support jinja templating that can be used to conditionally pass state or pass in env variables through the command line via the `--config-var` option.
+
+*./worker.yaml*
+```yaml
+terraform:
+  providers:
+    aws:
+      vars:
+        region: {{ aws-region }}
```
Comment (jessewiles) [inline README.md:75]: this should be {{ aws_region }} i discovered recently... :)

File: README.md [hunk 2]
```diff
@@ -62,6 +62,52 @@ Build and publish the package to PYPI:
...
+        version: "~> 2.61"
```
Comment (jessewiles) [inline README.md:76]: also, at the present moment, we need the minor.minor version number. (2.16.0) ?

**Review by jessewiles — APPROVED** [review]
Couple of suggestions. :D

**Comment by ephur** [comment]
going to merge this and address jwiles comments since nick has left, don't want this to hang out forever

**Comment by nrfox** [comment]
> going to merge this and address jwiles comments since nick has left, don't want this to hang out forever

I'm happy to make the changes I just won't be able to merge it anymore :)

**Pull request merged**